.travis.yml

```diff
@@ -91,59 +81,77 @@ jobs:
         - npm install
         - npm run build
 
+      before_deploy:
+        - ~/.local/bin/aws s3 rm s3://$AWS_S3_BUCKET_PROD$DEPLOY_PATH --recursive --region $AWS_S3_REGION
+
       deploy:
-        provider: script
-        script: ~/.local/bin/aws s3 sync dist s3://$AWS_S3_BUCKET_PROD/kuzzle-v1 --delete
+        provider: s3
+        access_key_id: $AWS_ACCESS_KEY_ID
+        secret_access_key: $AWS_SECRET_ACCESS_KEY
+        bucket: $AWS_S3_BUCKET_PROD
         skip_cleanup: true
+        local-dir: dist
+        region: $AWS_S3_REGION
+        on:
+          all_branches: true
 
       after_deploy:
-        - aws cloudfront create-invalidation --distribution-id $AWS_CLOUDFRONT_DISTRIBUTION_ID_PROD --paths "/kuzzle-v1/*"
+        - ~/.local/bin/aws cloudfront create-invalidation --distribution-id $AWS_CLOUDFRONT_DISTRIBUTION_ID_PROD --paths "$DEPLOY_PATH*"
 
-# ---------------------------------------
-# next.console.kuzzle.io
-# ---------------------------------------
+    # ---------------------------------------
+    # next.console.kuzzle.io
+    # ---------------------------------------
     - stage: build & deploy
-      name: "Build & Deploy to next.console.kuzzle.io"
+      name: 'Build & Deploy to next-console.kuzzle.io'
 
       if: type != pull_request && branch = 3-dev
 
+      env:
+        - DEPLOY_PATH=/kuzzle-v1/
```

Comment:
```suggestion
        - DEPLOY_PATH=/kuzzle-v2/
```

Reply:
err... admin console v2 is not for kuzzle v1?

Reply:
yep, but this is for the job that deploys `3-dev`. Currently there's no job in this file that deploys 2-stable nor 2-dev.